core/app/models/spree/tax/item_adjuster.rb

```diff
@@ -0,0 +1,32 @@
+module Spree
+  module Tax
+    class ItemAdjuster
+      include TaxHelpers
+
+      def initialize(item, options = {})
+        @item = item
+        @order = @item.order
```

Comment:
`@item` and `@order` should be `attr_reader`s, so the `TaxHelper` `module` and potential sub classes don't rely on reading instance variables.